### ISSUE TITLE
Please add (newby) me - Seattle

### ISSUE THREAD

**Issue by geekgurl-muraco** [githubschool/open-enrollment-classes-introduction-to-github#7456]
Hello! :wave: Please add me as a collaborator to this repo. 

@geekgurl-muraco

**Comment by githubteacher** [comment]
:tada: @geekgurl-muraco you are now a repository collaborator. :balloon:

**Comment by geekgurl-muraco** [comment]
geekgurl-muraco is mostly lost - or should it be @geekgurl-muraco ? I have watched 2 hours of githelp but still lost... I admit to being a newby also to coding... looking for other instructional videos as what I am here seems to make assumptions about what I should know already. Any assistance to learning github would be appreciated. Believe it not I have advanced degrees but not in coding. 
For example should I be writing geekgurl-muraco is lost or @geekgurl-muraco is lost? LOL...
